src/arrayMethodSort.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const temp = this[i];
+
+          this[i] = this[j];
+          this[j] = temp;
+        }
+      }
+    }
```

Comment:
While the bubble sort algorithm is correctly implemented, it is known to be one of the least efficient sorting algorithms for larger datasets. If performance is a concern, consider implementing a more efficient algorithm such as quicksort or mergesort.